ci/generate-workflows.py

```diff
@@ -130,7 +130,7 @@ def __init__(self, name, path, key):
 
 class CacheRustStep(ActionStep):
     def __init__(self, name, key):
-        super().__init__(name, action="Swatinem/rust-cache@v2", params={"key": key})
+        super().__init__(name, action="mozilla-actions/sccache-action@v0.0.3")
```

Comment:
`sccache` has no way of adding a custom key yet(?). Might or might not trip us up